tools/plugin/modules/alsa.c

```diff
@@ -266,23 +266,7 @@ static int set_params(struct comp_dev *dev)
 		return err;
```

Comment:
Little light on the git commit on why capture and playback are handled differently.